packages/now-next/src/utils.ts

```diff
@@ -565,6 +571,28 @@ export async function getPrerenderManifest(
   }
 }
 
+async function getSourceFilePathFromPage({
+  workPath,
+  page,
+  pagesDir,
+}: {
+  workPath: string;
+  page: string;
+  pagesDir: string;
+}) {
+  const source = path.join(workPath, 'src', 'pages');
+
+  try {
+    if ((await fs.stat(source)).isDirectory()) {
```

Comment:
I think you can move this stat call out of the loop because it only needs to happen once per deployment instead of once per page in the deployment, right?

Reply:
I've made it so that it is cached inside a variable, so the `stat` call is only made once